pyaedt/modules/Boundary.py

```diff
@@ -1607,29 +1623,96 @@ def join_parallel(self, sources, matrix_name=None, join_name=None):
 
 
 class MaxwellMatrix(object):
-    def __init__(self, app, parent_name):
+    def __init__(self, app, parent_name, reduced_name):
         self._app = app
-        self._parent_matrix = parent_name
-        self.names = []
-        self.sources = []
+        self.parent_matrix = parent_name
+        self.name = reduced_name
+        self.__sources = None
+
+    @property
+    def sources(self):
+        """List of matrix sources.
```

Comment:
Generally, a property doesn't have "Parameters" and "Returns" Sections. You can delete them.